pkg/pillar/types/zedroutertypes.go

```diff
@@ -782,6 +782,53 @@ const (
 	MST_LAST = 255
 )
 
+// IntfClass - enum definition for device Interface Class
+type IntfClass uint8
+
+// enum ddfinition for interface classes
```

Comment:
I don't think we should define classes but instead add a numeric priority in the API. For now you can use the freeUplink which is in DeviceNetworkStatus; later we'll add  the numeric priority end to end.

Reply:
Ok, will modify that part of the patch.